docs/src/pages/components/checkboxes/Checkboxes.js

```diff
@@ -1,13 +1,15 @@
 import * as React from 'react';
 import Checkbox from '@material-ui/core/Checkbox';
 
+const ariaLabel = { inputProps: { 'aria-label': 'Checkbox demo' } };
+
 export default function Checkboxes() {
   return (
     <div>
-      <Checkbox defaultChecked inputProps={{ 'aria-label': 'checked' }} />
-      <Checkbox inputProps={{ 'aria-label': 'uncontrolled' }} />
-      <Checkbox disabled inputProps={{ 'aria-label': 'disabled' }} />
-      <Checkbox disabled checked inputProps={{ 'aria-label': 'disabled checked' }} />
+      <Checkbox {...ariaLabel} defaultChecked />
```

Comment:
```suggestion
      <Checkbox {...ariaLabel} defaultChecked />
```
Just `label` is fine.

Reply:
OK, let's do this